docs/Privacy.md

```diff
@@ -17,4 +17,5 @@ Currently telemetry is only implemented for Windows builds and is turned **ON**
 The Windows provider uses the [TraceLogging](https://docs.microsoft.com/en-us/windows/win32/tracelogging/trace-logging-about) API for its implementation. This enables ONNX Runtime trace events to be collected by the operating system, and based on user consent, this data may be periodically sent to Microsoft servers following GDPR and privacy regulations for anonymity and data access controls. 
 
 Windows ML and onnxruntime C APIs allow Trace Logging to be turned on/off (see [API pages](../README.md#api-documentation) for details).
-For information on how to enable and disable telemetry, see [C API: Telemetry](./C_API.md#telemetry).
+For information on how to enable and disable telemetry, see [C API: Telemetry](./C_API.md#telemetry). 
+There are equivalent APIs in the C# and Python language bindings as well.
```

Comment:
We've got the same hook in the Java API too -  https://github.com/microsoft/onnxruntime/blob/master/java/src/main/java/ai/onnxruntime/OrtEnvironment.java#L218

Reply:
Updated the doc to include Java as well, thanks